src/unix/bsd/netbsdlike/openbsd/mod.rs

```diff
@@ -1576,6 +1610,9 @@ pub const NTFS_MFLAG_ALLNAMES: ::c_int = 0x2;
 
 pub const TMPFS_ARGS_VERSION: ::c_int = 1;
 
+pub const SI_MAXSZ: ::size_t = 128;
+pub const SI_PAD: ::size_t = (SI_MAXSZ / ::mem::size_of::<::c_int>()) - 3;
+
```

Comment:
I would not make them public: there are not used outside the struct definition anywhere inside OpenBSD tree.

Reply:
Hmmm I would object at least for SI_PAD at least don't you think ?

Reply:
it is simpler to make it public if need than to hide it later.

but even in C, it wouldn't be needed: it is the size of the padding.
you might need the size of the struct, but not just of a part of the struct (and you could still use `offsetof()` macro). 

here, the "struct" is hidden inside a function. so for me there is really no point to have it public (even if it isn't wrong per se). it is just an internal, or an implementation detail.

Reply:
good point about the struct being within these accessors, ok I ll commit new change.